04-store-api/starter/app.js

```diff
@@ -1 +1,41 @@
-console.log('04 Store API')
+require('dotenv').config()
+require('express-async-errors')
+
+const express = require('express');
+const app = express();
+
+const connectDB = require('./db/connect')
+const productsRouter = require('./routes/products')
+
+const notFoundMiddleware = require('./middleware/not-found')
+const errorMiddleware = require('./middleware/error-handler')
+
+//middleware
+app.use(express.json())
+
+//routes
+
+app.get('/', (req,res) => {
+    res.send('<h1> Store API</h1> <a href="/api/v1/products">products route</a>')
+})
+
+app.use('/api/v1/products', productsRouter)
+
+//products route
+
+app.use(notFoundMiddleware)
+app.use(errorMiddleware)
```

Comment:
I don't know if you were struggling to hunt down the errors caused by the syntax typo, but temporarily commenting out the root error handler will cause your app to crash loudly when an error happens, which might make it easier to trace, and can be a helpful debugging technique with these express projects.